tsconfig.typedoc.json

```
@@ -76,7 +76,7 @@
       "packages/test-util/src/index.ts",
       "packages/temporal-de-serializers/src/index.ts"
     ],
-    "name": "SAP Cloud SDK for JavaScript / TypeScript",
+    "name": "SAP Cloud SDK for JavaScript (and TypeScript)",
```

```suggestion
    "name": "SAP Cloud SDK for JavaScript",
```
[pp] Like Florian mentioned, the official name is SAP Cloud SDK for JavaScript. We can just leave out the TS part.